gprofiler/merge.py

```diff
@@ -250,9 +252,11 @@ def concatenate_profiles(
 
     for pid, stacks in process_profiles.items():
         container_name = _get_container_name(pid, docker_client, add_container_names)
+        application_name = get_application_name(pid) or "" if add_application_names else ""
+        prefix = container_name + ";" if add_container_names else "" + application_name
```

Comment:
Order of operators here is not clear (on the left hand there's `container_name + ";"` that are in the `if`, and o the right side there is `"" + application_name` where you clearly meant that `""` is in the `if` and `application_name` is not. But either both are or both are not. Please fix and add parenthesis

Reply:
Not relevant anymore..